### PR TITLE
Beta feature directive

### PR DESCRIPTION
**Pull request by sergeysenja1992**

### CODE REVIEW

File: packages/core/config/src/xm-beta-feature.directive.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+export interface XmBetaFeatureConfig extends XmUIConfig {
+    betaFeaturesEnabled: boolean;
+}
+
+/**
+ * Conditionally includes an HTML element if beta features enabled
+ */
+@Directive({
+    selector: '[xm-beta-feature]'
+})
```
Comment (Kaskyi) [inline packages/core/config/src/xm-beta-feature.directive.ts:14]: standalone: true,

**Review by Kaskyi — APPROVED** [review]

File: packages/core/config/src/xm-beta-feature.directive.ts (hunk 1 above)
Comment (sergeysenja1992) [reply packages/core/config/src/xm-beta-feature.directive.ts:14]: I registered it in module

File: packages/administration/dashboards-config/components/widget-edit/widget-edit.component.ts [hunk 2]
```diff
@@ -225,4 +225,8 @@ export class WidgetEditComponent implements OnChanges {
 
         this.value = _.merge(this.value, copiedObject.config);
     }
+
+    private get dashboardId(): number {
+        return this.value?.dashboard?.id || this.value?.dashboard;
```
[the remark this answers is not in the document]
Comment (sergeysenja1992) [reply packages/administration/dashboards-config/components/widget-edit/widget-edit.component.ts:230]: this.value?.dashboard - contains id after save widget, and contains object with id on open

[the remark this answers is not in the document]
Comment (sergeysenja1992) [reply packages/administration/dashboards-config/components/widget-edit/widget-edit.component.ts:230]: this is ugly widget, and maybe one day I will refactor it

[the remark this answers is not in the document]
Comment (sergeysenja1992) [reply packages/administration/dashboards-config/components/widget-edit/widget-edit.component.ts]: fixed

File: packages/core/config/src/xm-beta-feature.directive.ts (hunk 1 above)
Comment (sergeysenja1992) [reply packages/core/config/src/xm-beta-feature.directive.ts:15]: fixed

File: packages/administration/dashboards-config/configuration-history/configuration-history.component.ts [hunk 3]
```diff
@@ -12,11 +12,12 @@ import { DASHBOARDS_TRANSLATES } from '../const';
 import { XmTranslationModule } from '@xm-ngx/translation';
 import { MatBadgeModule } from '@angular/material/badge';
 import { DialogModule } from '@angular/cdk/dialog';
+import { XmBetaFeatureDirective } from "@xm-ngx/core/config/src/xm-beta-feature.directive";
```
Comment (Kaskyi) [inline packages/administration/dashboards-config/configuration-history/configuration-history.component.ts:15]: use `  '@xm-ngx/core/config'` for packages

 add export to `src/index.ts`
export {XmBetaFeatureDirective} from './xm-beta-feature.directive

File: packages/core/config/src/xm-core-config.module.ts [hunk 4]
```diff
@@ -1,11 +1,13 @@
 import { NgModule } from '@angular/core';
 import { XmPrivateUiConfigService } from './xm-private-ui-config.service';
 import { XmUiConfigService } from './xm-ui-config.service';
+import { CommonModule } from "@angular/common";
 
 @NgModule({
     providers: [
         XmPrivateUiConfigService,
         XmUiConfigService,
+        CommonModule
```
Comment (Kaskyi) [inline packages/core/config/src/xm-core-config.module.ts:10]: no need any more, we can remove it

**Review by Kaskyi — APPROVED** [review]

File: packages/administration/dashboards-config/configuration-history/configuration-history.component.ts (hunk 3 above)
Comment (sergeysenja1992) [reply packages/administration/dashboards-config/configuration-history/configuration-history.component.ts]: fixed

File: packages/core/config/src/xm-core-config.module.ts (hunk 4 above)
Comment (sergeysenja1992) [reply packages/core/config/src/xm-core-config.module.ts]: fixed

**Review by denisogolubev — APPROVED** [review]

**Pull request merged**